### DISCUSSION TITLE
Doubt about Fixtures usage on Deployment in Tests

### DISCUSSION THREAD

**Discussion by afa7789** [PaulRBerg/hardhat-template#179] [Q&A] [answered]
@PaulRBerg , hey paul

this is my first time using this template of yours ( for a internal tool that will be public ), before typescript I could do some stuff more freely on my deployers and test, now trying to follow your code base I could not make a deployment in the test that receives variables do you have an opinion or answer on how to do it here:

<img width="734" alt="image" src="https://github.com/PaulRBerg/hardhat-template/assets/26887703/8939140e-2517-4350-83bf-cd1be99b38a5">

<img width="619" alt="image" src="https://github.com/PaulRBerg/hardhat-template/assets/26887703/403f7b81-564f-44d0-850b-aaa6905b501e">

**Comment by PaulRBerg** [comment, marked as answer]
Hello @afa7789! Thanks for your question.

Unfortunately, I am no longer using this template on a regular basis (I am using [Foundry](https://github.com/PaulRBerg/foundry-template) now), so I don't know how to answer your question. You might have better luck looking for help in the [Hardhat Discord server](https://discord.com/invite/TETZs2KK4k).
